rapids-cmake/cython/create_modules.cmake

```diff
@@ -0,0 +1,69 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+#[=======================================================================[.rst:
+rapids_cython_create_modules
+---------------------
+
+.. versionadded:: v22.06.00
+
+Generate C(++) from Cython and create Python modules.
+
+.. code-block:: cmake
+
+  rapids_cython_create_modules(<ModuleName...>)
+
+Creates a Cython target for a module, then adds a corresponding Python
+extension module. This function must be called after rapids_cython_init.
+
+``MODULES``
+  The list of modules to build.
```

Comment:
We should clarify what modules means. Maybe `This list of sources to build. Each source will be converted to a unique cython module"

Reply:
Regarding terminology: we might want to consistently use "extension modules"  to refer to the `.so` files resulting from the compilation of  `.pyx` files ("Cython source files").

Reply:
Yes, I think this is the right approach. We should consistently use "Python extension module" for Python libraries we are building to differentiate clearly from CMake modules (or pure Python files for that matter).